### PR TITLE
Added basic functionality to the BOT

### PR DESCRIPTION
**Pull request by DefCon-007**
To run it few more environment variables will be needed. Right now it is hosted on my heroku account. Callback url : evening-ravine-92836.herokuapp.com
Bot url : m.me/1880474155521998/
To test it out ping me (@defcon) on slack with your fb username and I will add you as a developer.

cc : @AvijitGhosh82

### CODE REVIEW

**Comment by kshitij10496** [comment]
Nice work here @DefCon-007 
It would be helpful for everyone if you can document it in the README.

File: SO_scrapper.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import requests
+from bs4 import BeautifulSoup as bs
```
Comment (kshitij10496) [inline SO_scrapper.py]: The Stack Exchange family has an API which we can use here instead of scrapping.

**Comment by kshitij10496** [comment]
Awesome work here @DefCon-007 
This is in.

**Comment by OrkoHunter** [comment]
Cheers @DefCon-007 :tada:

**Pull request merged**